Orange/widgets/data/owselectrows.py

```diff
@@ -73,7 +73,7 @@ def __init__(self):
         self.cond_list = QtGui.QTableWidget(
             box, showGrid=False, selectionMode=QtGui.QTableWidget.NoSelection)
         box.layout().addWidget(self.cond_list)
-        self.cond_list.setColumnCount(3)
+        self.cond_list.setColumnCount(4)
```

Comment:
They aren't required, they are all optional. But would look much better imho.

Thanks for working on this!